build.gradle

```diff
@@ -1003,7 +1003,7 @@ def calculateVersion() {
     return "${project.version}"
   } else {
     // If no version is supplied or it doesn't match the semantic versioning, calculate from git
-    println("Generating project version as supplied is version not semver: ${project.version}")
```

Comment:
seems like an unintended change?

Reply:
possibly a rebase issue.  thanks for catching